README.md

```diff
@@ -22,25 +22,53 @@ After installing, you need to require Composer's autoloader:
 require 'vendor/autoload.php';
 ```
 
+## Setting up a Request Adapter
```

Comment:
I would put an example composer.json file here too so people can just grab it if they want to use these deps, ie

```json
{
  "require": {
    "sparkpost/php-sparkpost": "~1.0",
    "egeloen/http-adapter": "0.8.0",
    "guzzlehttp/guzzle": "6.1.0",
    "php-http/guzzle6-adapter": "0.1.0"
  }
}
```